overrides.json

```
@@ -35,6 +35,25 @@
       }
     }
   },
+  "@material-ui/lab": {
+    "4": {
+      "exports": {
+        ".": "./index.js",
```

Should this also be something like:

```suggestion
        ".": {
          "require": "./index.js",
          "import": "./esm/index.js"
        }
```